release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -3332,3 +3332,21 @@ link:https://access.redhat.com/solutions/6962609[{product-title} 4.7.52 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-7-53"]
+=== RHSA-2022:4965 - {product-title} 4.7.53 bug fix and security update
+
+Issued: 2022-06-16
+
+{product-title} release 4.7.53 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:4965[RHSA-2022:4965] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:4965[RHSA-2022:4965] advisory.
+
+There are no images for {product-title} 4.7.53.
+
+[id="ocp-4-7-53-bug-fixes"]
+==== Bug fixes
+* A vulnerability is present in CRI-O that causes memory exhaustion on the node for anyone with access to the Kubernetes API. When ExecSync request runs commands in a container and logs the output, large output files read by cri-o after execution can possibly exhaust the memory of the node.
```

Comment:
When the ExecSync? The intro of this sentence is a little confusing to me.

CRI-O after